packages/web/src/components/withCellHOC.tsx

```diff
@@ -70,6 +72,24 @@ export interface WithCellProps {
  *  return <div><ExampleComponent /></div>
  * }
  */
+
```

Comment:
No reason to have this in the render block, right? Moved this up

Reply:
Yeah. I don't really like this little utility checks myself because I find this code hard to read.